resources/assets/less/app.less

```diff
@@ -1,64 +1,90 @@
-html {
-    box-sizing: border-box;
-}
-*, *:before, *:after {
-    box-sizing: inherit;
-}
-
-@import "vendor/bootstrap/bootstrap";
-@import "utilities";
-@import "github";
-@import "components/gistlog";
-@import "components/comment";
-@import "components/notice";
+@import "./tailwind";
 
-@btn-font-weight: 300;
-@font-family-sans-serif: "Roboto", Helvetica, Arial, sans-serif;
-@font-family-monospace: Consolas, Menlo, Monaco, "Courier New", monospace;
+@import "./github.less";
 
-/**
- * Bootstrap-esque
- */
+html {
+    @apply .font-sans;
+}
 
-/**
- * Custom
- */
+body {
+    @apply .mx-auto .bg-grey-lightest;
+    max-width: 1440px;
+}
 
-body, label, .checkbox label {
-    font-weight: 300;
+.avatar {
+    @apply .rounded-full .overflow-hidden .relative .mx-auto;
+    width: 5.5rem;
+    height: 5.5rem;
 }
 
-img {
-    max-width: 100%;
+.btn-github {
+    @apply .flex .bg-grey-darkest .text-sm .px-3 .pt-1 .rounded .my-4;
+    color: #fff !important;
 }
 
-.logo {
-    height: 1.2em;
+.gistlog {
+    @apply .bg-white .shadow  .py-8;
+    margin-top: -2.6rem;
```

Comment:
Hm, what is this negative top margin offsetting? Usually (not always) negative margins show a sign of something being wrong elsewhere.